styleguide.config.js

```diff
@@ -44,13 +57,14 @@ module.exports = {
   ],
   webpackConfig: {
```

Comment:
@diversemix Is your comment suggesting that we move this piece of the styleguidist config out of this file, and put it under the `/webpack` directory?

Reply:
My question is, if it is webpack config - why doesn't it live in the webpack folder?

Reply:
The webpack portion of this config file does now live in the webpack folder. But the rest of the styleguidist config is still in the root.

Let us know what you think after this latest commit :+1: